static/app/views/detectors/components/details/cron/index.tsx

```diff
@@ -228,6 +230,11 @@ export function CronDetectorDetails({detector, project}: CronDetectorDetailsProp
                 showUnknownLegend={showUnknownLegend}
               />
             </Section>
+            {detector.description && (
+              <Section title={t('Describe')}>
```

Comment:
Should it say "description" instead of describe? the preview resolve/assign/etc labels are more about what the monitor is _doing_, whereas this is a meta property of the monitor